src/main/java/graphql/schema/TypeVisitorStub.java

```diff
@@ -0,0 +1,76 @@
+package graphql.schema;
+
+import graphql.util.TraversalControl;
+import graphql.util.TraverserContext;
+import static graphql.util.TraversalControl.CONTINUE;
+
+public class TypeVisitorStub implements TypeVisitor {
```

Comment:
Simple JavaDoc please.

This probably deserves to be @PublicApi eventually (as a help to write a visitor)

Reply:
Also naming as mentioned above `GraphqlTypeVisitorStub`